staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -966,6 +971,14 @@ type MemorySwapConfiguration struct {
 	SwapBehavior string `json:"swapBehavior,omitempty"`
 }
 
+type CrashLoopBackOffConfig struct {
+	// maximumBackOffPeriod is the maximum duration the backoff delay can accrue
+	// to for container restarts, minimum 1 second, maximum 300 seconds.
+	// +featureGate=KubeletCrashLoopBackOffMax
+	// +optional
```

Comment:
Still for discussion: default

My preference would be to state the default here, and add it to defaults.go.

Reply:
👍  i'm taking a look in a new commit

Reply:
specify the default?